src/logging.h

```diff
@@ -210,7 +215,7 @@ static inline void LogPrintf_(const std::string& logging_function, const std::st
 
 #define LogPrintLevel(level, category, ...)               \
     do {                                                  \
-        if (LogAcceptCategory((category))) {              \
+        if (LogAcceptCategory((category, level))) {       \
```

Comment:
At first glance it looks like the callers are using an inverted parameters order (level, category) compared to here.

Reply:
I thought about that, and I'm fine with putting level first, but that means updating all callers of `LogAcceptCategory`. It's not that many, though.

Reply:
It should build correctly now, the problem wasn't the order of arguments but invalid `()` nesting.

Reply:
I agree it's strange that `LogPrintLevel` and `LogPrintLevel_`/`LogAcceptCategory` have inverted order of level, category arguments, but not sure it's in scope to fix here (nor in which direction to resolve it, I think that needs a separate discussion).